msprime/mutations.py

```diff
@@ -307,58 +307,75 @@ class MicrosatMutationModel(MatrixMutationModel):
     """
     General class for Microsatellite mutation models
     which parameterizes the transition matrix describing
-    changes among alleles using 5 parameters according
-    to the model of Sainudiin et al. (2004).
+    changes among alleles using five parameters according
+    to the model of
+    `Sainudiin et al. (2004) <https://doi.org/10.1534/genetics.103.022665>`_
 
     Concretely, mutation rates under this model are computed as follows.
     Let ``qij`` be the transition rate from equation (3) in Sainudiin et al,
     and let ``M`` be the largest row sum of the matrix whose i,j-th entry
     is ``qij`` and whose diagonals are zero. Then if the mutation rate
     passed to :func:`.sim_mutations` is ``m``, the mutation rate from
-    allele ``i`` to allele `j`` is ``m * qij / M``.
+    allele ``i`` to allele ``j`` is ``m * qij / M``.
 
-    :param float s: strength of length dependence on mutation rate
-        (i.e. s=0 uniform rate)
-    :param float u: constant bias parameter; must be between 0 and 1
-    :param float v: linear bias parameter
-    :param float p: probability of a single step mutation
-    :param float m: 1 / (mean multistep mutation size)
-    :param int lo: lower bound on repeat number
-    :param int hi: upper bound on repeat number
+    Note that the values of ``lo`` and ``hi`` are in units of repeat number,
+    and are unrelated to repeat length
+    (e.g., the repeat itself can be dinucleotide, tri-nucleotide, etcetera).
+
+    The default values for ``lo`` and ``hi`` are chosen based on
+    FIXME.
```

Comment:
so these aren't based on anything in particular-- defaults for `lo` and `hi` are just for convenience